app/controllers/redirect/contact_controller.rb

```diff
@@ -0,0 +1,7 @@
+module Redirect
+  class ContactController < RedirectController
+    def show
+      redirect_to_and_log IdentityConfig.store.idv_contact_url
```

Comment:
can we add a `tracker_method:` or specific event to track for this redirect? Would we also want track that it's specifically for the review workflow?